pkg/syncer/k8scloudoperator/k8scloudoperator.go

```diff
@@ -305,6 +307,15 @@ func (k8sCloudOperator *k8sCloudOperator) PlacePersistenceVolumeClaim(ctx contex
 		return out, nil
 	}
 
+	// Abort placement if the storage class has Immediate volume binding and nodeAffinity PVC annotation is not specified
```

Comment:
Can you please add this scenario to the testing done section?

Reply:
> A question on the testing done. In the 3rd test, how did you force Pod placement to sc2-10-186-82-158.eng.vmware.com? Was this done through Pod's nodeAffinity annotation?
No, I didn't do anything specific to select this node, it's selected by the scheduling algorithm of DRS. 

> Also, once this change is done, we could also clean up the external provisioner to ignore the custom PVC nodeAffinity annotation, right?
Yes, I've created an internal MR in gitlab in external provisioner and I'll add you for the review.

> Can you please add this scenario to the testing done section?
Done